its/src/test/java/com/sonar/it/scanner/msbuild/ScannerMSBuildTest.java

```diff
@@ -77,6 +81,42 @@ public void testSample() throws Exception {
     assertThat(getProjectMeasure("ncloc").getIntValue()).isEqualTo(37);
     assertThat(getFileMeasure("lines").getIntValue()).isEqualTo(58);
   }
+  
+  @Test
+  public void testMultiLanguage() throws Exception {
+    ORCHESTRATOR.getServer().restoreProfile(FileLocation.of("projects/ConsoleMultiLanguage/TestQualityProfileCSharp.xml"));
+    ORCHESTRATOR.getServer().restoreProfile(FileLocation.of("projects/ConsoleMultiLanguage/TestQualityProfileVBNet.xml"));
+    ORCHESTRATOR.getServer().provisionProject(PROJECT_KEY, "multilang");
+    ORCHESTRATOR.getServer().associateProjectToQualityProfile(PROJECT_KEY, "cs", "ProfileForTestCSharp");
+    ORCHESTRATOR.getServer().associateProjectToQualityProfile(PROJECT_KEY, "vbnet", "ProfileForTestVBNet");
+
+    Path projectDir = TestUtils.projectDir(temp, "ConsoleMultiLanguage");
+    ORCHESTRATOR.executeBuild(TestUtils.newScanner(projectDir)
+      .addArgument("begin")
+      .setProjectKey(PROJECT_KEY)
+      .setProjectName("multilang")
+      .setProjectVersion("1.0")
+      .setDebugLogs(true));
+
+    TestUtils.runMSBuild(ORCHESTRATOR, projectDir, "/t:Rebuild");
+
+    ORCHESTRATOR.executeBuild(TestUtils.newScanner(projectDir)
+      .addArgument("end"));
+
+    List<Issue> issues = ORCHESTRATOR.getServer().wsClient().issueClient().find(IssueQuery.create()).list();
+    // 4 CS, 1 VBNET
+    assertThat(issues).hasSize(5);
+    
+    // FxCop rules do not show up for VB.NET because that version of the plugin no longer contains them (moved to FxCop plugin)
+    List<String> keys = issues.stream().map(i -> i.ruleKey()).collect(Collectors.toList());
+    assertThat(keys).containsAll(Arrays.asList("vbnet:S3385", 
+      "fxcop:DoNotRaiseReservedExceptionTypes", 
```

Comment:
No. That's how it is hardcoded in the Scanner for MSBuild (and apparently in the C# plugin).